CDP4SiteDirectory/ViewModels/Dialogs/PersonDialogViewModel.cs

```diff
@@ -92,6 +92,19 @@ public PersonDialogViewModel(Person person, ThingTransaction transaction, ISessi
                 this.RaisePropertyChanged("Password");
                 this.RaisePropertyChanged("PasswordConfirmation");
             });
+
+            this.WhenAnyValue(vm => vm.SelectedRole).Subscribe(x =>
+            {
+                this.RaisePropertyChanged(nameof(this.IsSelectedRoleDeprecated));
```

Comment:
I understand people here type property name as string, but I prefer it strongly typed to avoid typos.

Reply:
The explicit use of RaisPropertyChanged is a bit of an anti-pattern in ReactiveUI, typically the following pattern is used "ObservableAsPropertyHelpers ", read about it here: https://www.reactiveui.net/docs/guidelines/framework/prefer-oaph-over-properties

You will find (i hope) very little times that we explicitly call `RaisPropertyChanged`